src/app/courses/add-course/add-course.component.ts

```diff
@@ -2,7 +2,8 @@ import { Component, Input, OnInit, } from '@angular/core';
 import { Router } from '@angular/router';
 
 import { CourseService } from 'src/app/shared/services/course.service';
-import { Course } from 'src/app/shared/interfaces/course.interface.';
+import { HttpCourse } from 'src/app/shared/interfaces/http-course.interface';
+import { HttpAuthor } from 'src/app/shared/interfaces/http-author.interface';
```

Comment:
Try not to use imports from the src folder directly

Reply:
ok, made the import with relative path